qiskit_nature/circuit/library/ansatzes/chc.py

```diff
@@ -0,0 +1,312 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Compact heuristic ansatz for Chemistry """
```

Comment:
Is it a general chemistry Ansatz or is more meant for Vibrational problem calculations? Also tagging @paulineollitrault

Reply:
it's meant for Vibrational problems

Reply:
I will update the docstring in a second :+1: